src/main/core/Subscriber/SecurityEventSubscriber.php

```diff
@@ -62,6 +63,10 @@ public function logEvent(Event $event, string $eventName): void
         $logEntry->setTarget($event->getUser());
         $logEntry->setDoer($this->security->getUser());
 
+        if ($event instanceof AuthenticationFailureEvent) {
```

Comment:
You should allow the event to override the doer to handle this properly.

Reply:
After discussion no need for change